inc/azure_macro_utils/macro_utils.h

```diff
@@ -239,17 +237,27 @@ const char* MU_C3(MU_, enumIdentifier,_ToString)(enumIdentifier value)
 #define CONSTRUCT_FROM_FAKE_ENUM(from_value, to_value) \
     MU_C3(fake_from_, from_value, _C0FB3709_EDE8_4288_8F70_FDDB5D8D7A51),
 
+#define CONSTRUCT_FIELD_FROM_FAKE_ENUM(from_value, to_value) \
+    int MU_C3(fake_from_, from_value, _C0FB3709_EDE8_4288_8F70_FDDB5D8D7A51)[(MU_C2(enum_value_typedef_, from_value) == MU_C2(enum_value_typedef_, from_value)) ? 1 : 0];
+
+//    int MU_C3(fake_from_, from_value, _C0FB3709_EDE8_4288_8F70_FDDB5D8D7A51)[(MU_C2(enum_value_typedef_, from_value) == MU_C2(enum_value_typedef_, from_value)) ? 1 : 0];
```

Comment:
extra comment can be removed

Reply:
Removed.

---
In reply to: [396524885](https://github.com/Azure/azure-macro-utils-c/pull/37#discussion_r396524885) [](ancestors = 396524885)